src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -195,13 +196,13 @@ export default {
           // TODO: check if response is not null otherwise show an error
           
           const estimatedGas = new BigNumber(this.responseEstimatedGas?.amount)
-            .shiftedBy(-8)
+            .shiftedBy(decimals)
             .toString()
 
           this.sharedState.web3.eth.getGasPrice().then(gasPrice => {
             const costInWei = new BigNumber(estimatedGas)
               .multipliedBy(gasPrice)
-              .shiftedBy(-10)
+              .shiftedBy(decimals)
```

Comment:
The -10 was incorrect here right?

Reply:
Nope this isn’t correct. The token to take the decimals is the origin token, not the destination one

Reply:
And the output token has 18 decimals in this case